### ISSUE TITLE
VNext

### ISSUE THREAD

**Issue by dlacaille-baytech** [exceptionless/Exceptionless#232]
Hello,
Do you plan to create a VNext client config extensions?

**Comment by niemyjski** [comment]
Yes, we have an open issue that is tracking it here: https://github.com/exceptionless/Exceptionless.Net/issues/28 We've been waiting for rc2 tooling before starting. We have been talking about this in our slack channel as well (please join us http://slack.exceptionless.com). With the recent decision of project.json and tooling we need to make sure we do the right work so we don't do something that won't be supported come end of June. 

I'm going to close this as it's a duplicate of https://github.com/exceptionless/Exceptionless.Net/issues/28. I do know there is a third party pull request out there for this right now and we are working with them :).

**Comment by niemyjski** [comment]
Also, you should be able to use exceptionless today if you target the full clr of asp.net core (if you can't please let me know).

**Comment by dlacaille-baytech** [comment]
Sounds like a plan. Sorry, I missed the other ticket, I searched for "vnext." Will try the approach and let you know.
